layouts/partials/tw/donate-prefill-lg.html

```diff
@@ -57,7 +57,7 @@
         m: 'monthly',
         y: 'yearly',
       }[this.freq];
-      return `https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&campaign=${campaign}&theme=${theme}&amount=${this.amount}&installmentPeriod=${period}&utm_source=spotlightpa.org&utm_medium=elections&utm_campaign=donate-prefill`;
+      return `https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&campaign={{ $donateCampaign }}&theme={{ $donateTheme }}&amount=${this.amount}&installmentPeriod=${period}&utm_source=spotlightpa.org&utm_medium=elections&utm_campaign=donate-prefill`;
```

Comment:
NBD but I try to avoid mixing two templating languages in one place if I can help it, so for this, my preferred solution is 

```html
<div
     data-ga-category="{{ $donateGaCategory }}"
     data-donate-campaign="{{ $donateCampaign }}"
     data-donate-theme="{{ $donateTheme }}"
```
...

```js
    return `https://checkout.fundjournalism.org/memberform?org_id=spotlightpa&campaign={ this.$el.dataset.donateCampaign }&theme={ this.$el.dataset.donateTheme }&amount=${this.amount}&installmentPeriod=${period}&utm_source=spotlightpa.org&utm_medium=elections&utm_campaign=donate-prefill`;
```

Reply:
This is updated with https://github.com/spotlightpa/poor-richard/pull/363/commits/9277310ec5883cfaedebbdbd67b5221287f979ba